src/AutoRest.CSharp/LowLevel/Output/ConvenienceMethod.cs

```diff
@@ -45,7 +45,7 @@ private record RequestContentParameterInfo(CodeWriterDeclaration ContentVariable
                 {
                     if (converter.ConvenienceSpread == null)
                     {
-                        var parameter = convenienceParameter.GetConversionFormattable(protocolParameter.Type);
+                        var parameter = convenienceParameter.GetConversionFormattable(protocolParameter.Type, RequestMediaTypes?.FirstOrDefault());
```

Comment:
This is common function to get convenience parameter for all kinds parameter, path, query, head and body. I think RequestMediaType is only affected the body parameter. Need to make sure it will break other kind of parameter.

Reply:
Why it would break others? Could you think of a scenario, since all tests are passed and no SDK change.